lighthouse-core/test/results/sample_v2.json

```diff
@@ -2631,25 +2626,15 @@
       "id": "aria-meter-name",
       "title": "ARIA `meter` elements have accessible names",
       "description": "When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. [Learn more](https://web.dev/aria-name/).",
-      "score": 1,
-      "scoreDisplayMode": "binary",
-      "details": {
-        "type": "table",
-        "headings": [],
-        "items": []
-      }
+      "score": null,
+      "scoreDisplayMode": "notApplicable"
     },
     "aria-progressbar-name": {
       "id": "aria-progressbar-name",
       "title": "ARIA `progressbar` elements have accessible names",
       "description": "When an element doesn't have an accessible name, screen readers announce it with a generic name, making it unusable for users who rely on screen readers. [Learn more](https://web.dev/aria-name/).",
-      "score": 1,
```

Comment:
It seems these were trivially passing w/ score 1 because `aria-progressbar-name` check was never found in the axe results, so when LH sees that it just spits out `score: 1`.


side note: https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/accessibility/axe-audit.js#L37 why is score 1 if this is marked n/a? Shouldn't we do `null`, because that's what the audit result will end up being anyway?

Reply:
Extra explanation: I scratched my head for a second, but in order for us to mark as notApplicable if the audit isn't "informative" we need to explicitly find it in axe's `inapplicable` set. 

> why is score 1 if this is marked n/a? Shouldn't we do null, because that's what the audit result will end up being anyway?

Yeah we should. My guess is that while authoring "not applicable" feels closest to "this is passing, don't worry about it" and then 1 gets used.